runtime/stdlib/contracts/test.cdc

```diff
@@ -95,6 +95,38 @@ pub contract Test {
         pub fun useConfiguration(_ configuration: Configuration) {
             self.backend.useConfiguration(configuration)
         }
+
+        /// Returns all the logs from the blockchain, up to the calling point.
+        ///
+        pub fun logs(): [String] {
+            return self.backend.logs()
+        }
+
+        /// Returns the service account of the blockchain. Can be used to sign
+        /// transactions with this account.
+        ///
+        pub fun serviceAccount(): Account {
+            return self.backend.serviceAccount()
+        }
+
+        /// Returns all events emitted from the blockchain.
+        ///
+        pub fun events(): [AnyStruct] {
+            return self.backend.events("")
+        }
+
+        /// Returns all events emitted from the blockchain,
+        /// filtered by type.
+        ///
+        pub fun eventsOfType(_ type: Type): [AnyStruct] {
+            return self.backend.events(type.identifier)
```

Comment:
Doesn't have to be in this PR (could be a follow-up PR), but maybe also change the `backend.events()` method to accept an optional `Type?` parameter, so we avoid converting from `string -> type -> string`. Implementation (cadence-tools) can decide to convert to string if needed, but probably don't have to do it here.

I suggested optional `Type?` so `nil` can be passed for retrieving all events.